cli/script/command-executor.ts

```diff
@@ -1038,6 +1038,13 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
                 .then((): void => {
                     log("Successfully released an update containing the \"" + command.package + "\" " + (isSingleFilePackage ? "file" : "directory") + " to the \"" + command.deploymentName + "\" deployment of the \"" + command.appName + "\" app.");
                 })
+                .catch((err: Error): void => {
+                    if (err.message === "Not Found") {
+                        err.message = "Release failed, the size of the upload may be too large.";
```

Comment:
200mb - we might tweak this from the server side in the future

Release failed - Given the cryptic nature of this server response, we are not 100% positive that the error is a result of the size limit, hence i added a generic "release failed" and weak language like "may be too large". Do u have a suggestion on how better we can word this?